src/templates/registration.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Register</h1>
+    <form method="post" action="" novalidate>
```

Comment:
Let's make `post` to be uppercased. Like that:
```html
    <form method="POST" action="" novalidate>
```

It's an unspoken agreement to write [http methods](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods) in uppercase everywhere it is possible.